configure

```diff
@@ -22,7 +22,7 @@ host='auto'
 gmp='auto'
 perf='no'
 
-ar='ar'
+ar="${AR}"
```

Comment:
Sorry, but we need  a reasonable default value here! The AR variable is empty in my default build environment...

Reply:
Sure, no problem, I've added a check to set `ar='ar'` if AR variable is empty